src/components/SimpleTimePicker/index.tsx

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import { Picker } from "react-mobile-style-picker";
+import "react-mobile-style-picker/dist/index.css";
+import dayjs from "dayjs";
+import customParseFormat from "dayjs/plugin/customParseFormat";
+
+import Drawer from "../Drawer";
+import "./index.css";
+
+interface ISimpleTimePickerProps {
+  isVisible?: boolean;
+  onChange?(t: string): void;
+}
+
+dayjs.extend(customParseFormat);
+
+const hours = Array.from({ length: 12 }, (_, i) => `${i + 1}`);
+
+const minutes = Array.from(
+  { length: 60 },
+  (_, i) => `${i < 10 ? "0" : ""}${i}`
+);
+
+const periods = ["AM", "PM"];
+
+const SimpleTimePicker = ({ isVisible, onChange }: ISimpleTimePickerProps) => {
+  const [selectedHour, setSelectedHour] = React.useState(hours[0]);
+  const [selectedMinute, setSelectedMinute] = React.useState(minutes[0]);
+  const [selectedPeriod, setSelectedPeriod] = React.useState(periods[0]);
```

Comment:
avoid using React.useState, you already import React on top add it there

Reply:
Updated